config/plugins.js

```diff
@@ -25,7 +25,17 @@ module.exports = {
 		'import/core-modules': [
 			'electron',
 			'atom'
-		]
+		],
+		'import/resolver': {
```

Comment:
Shouldn't it be as follow?
```
import/parsers: {
    typescript-eslint-parser: [ .ts, .tsx ]
}
```

Also should we add that only for `ts`/`tsx` files similarly to `parser = 'typescript-eslint-parser'`?

Reply:
Yep.

Reply:
Actually still need this or the import/unresolved rule throws.

Reply:
So it should be done dynamically in the `index.js` only for `ts`/`tsx` files I imagine

Reply:
I don't think it's an issue unless someone is intentionally mixing `.js` and `.ts` files in a directory.

Reply:
Could you move the `import/parsers` and `import/resolver` in `index.js` so it will be enabled only for `ts` and `tsx` files? This way that would avoid creating additional load and add a dependency to `typescript-eslint-parser` for non TS file.
```js
if (paths.filter(filePath => isTSFile(path.extname(filePath))).length > 0) {
  opts.parser = 'typescript-eslint-parser';
  opts['import/resolver'].node.extensions = ['.js', '.jsx', '.ts', 'tsx'];
  opts['import/parsers'].['typescript-eslint-parser'] = ['.ts', 'tsx'];
}
```

Reply:
Okey dokey